### ISSUE TITLE
False positive with injected service detection `foo.service()` in classic class

### ISSUE THREAD

**Issue by bmish** [ember-cli/eslint-plugin-ember#1190]
Our util `emberUtils.isInjectedServiceProp()` should be false for this: 

```js
import {inject as service} from '@ember/service';
export default Controller.extend({
  currentUser: foo.service()
});
```

This affects several service-related rules.

CC: @lin-ll
